src/main/java/com/github/manolo8/darkbot/core/manager/MapManager.java

```diff
@@ -455,6 +458,14 @@ public boolean isTarget(Entity entity) {
         return API.readMemoryLong(API.readMemoryLong(mapAddress + 120) + 40) == entity.address;
     }
 
+    public boolean setTarget(long entity) {
+        long targetWrapper = API.readLong(mapAddress + 120);
+        if (targetWrapper > 0xFFFF) {
```

Comment:
I've seen thin 0xFFFF constant used in quite a few places, maybe a ByteUtils.isValidPtr(targetWrapper) or smthg?